Removing inline styles from LoadingText

bobsilverberg opened the pull request:
Fixes #9511 

This should fix the csp errors by removing the inline styles. It also eliminates the `range` option to `LoadingText`, but I don't think we really needed that. I visually verified the elements that would have been affected, and their loading skeletons seem fine to me.

bobsilverberg commented:
@willdurand I assume you want me to hold off merging this until you complete your review?

willdurand commented:
> @willdurand I assume you want me to hold off merging this until you complete your review?

Right, I wanted to try this patch locally first. I noticed an overflow issue on this page:

- http://127.0.0.1:3000/fr/firefox/addon/frfrfr/reviews/?score=5 (title of the main "Reviews" card)

I couldn't reproduce on -dev.

bobsilverberg commented:
> > @willdurand I assume you want me to hold off merging this until you complete your review?
> 
> Right, I wanted to try this patch locally first. I noticed an overflow issue on this page:
> 
>     * http://127.0.0.1:3000/fr/firefox/addon/frfrfr/reviews/?score=5 (title of the main "Reviews" card)
> 
> 
> I couldn't reproduce on -dev.

I tried to check as many things as possible locally. I'll give than link a try locally.

bobsilverberg commented:
> > > @willdurand I assume you want me to hold off merging this until you complete your review?
> > 
> > 
> > Right, I wanted to try this patch locally first. I noticed an overflow issue on this page:
> > ```
> > * http://127.0.0.1:3000/fr/firefox/addon/frfrfr/reviews/?score=5 (title of the main "Reviews" card)
> > ```
> > 
> > 
> > I couldn't reproduce on -dev.
> 
> I tried to check as many things as possible locally. I'll give than link a try locally.

I don't see any overflow issues on that page. What specifically were you seeing?

willdurand commented:
> I don't see any overflow issues on that page. What specifically were you seeing?

It's maybe more obvious in RTL: http://127.0.0.1:3000/fa/firefox/addon/frfrfr/reviews/?score=1

![2020-09-29 16 52 04](https://user-images.githubusercontent.com/217628/94575510-9b4e6480-0274-11eb-94f5-261b8ae0bd4b.gif)

I simply change the selected value but it's not captured in the Gif above. We can see that I can easily trigger this problem two or three times.

bobsilverberg commented:
I realized the problem here is that there used to be a default where `60` would be the max width value if no props were passed to `LoadingText`, but I removed that. I added a `maxWidth` prop, which defaults to `60`, which brings back that functionality. When testing before adding that I saw some other places where there was overflow with RTL, but now everything looks good to me.
